### PR TITLE
Align RHEL 7 CIS control file with CIS v4.0.0 - Section 5

### PR DESCRIPTION
**Pull request by jan-cerny**
#### Description:

In this PR, we change the control file, change references, add existing rules. But, we don't add new rules.

#### Rationale:
Align RHEL 7 CIS control file with CIS v4.0.0

### CODE REVIEW

**Comment by jan-cerny** [comment]
The CI fail on Rawhide is caused by https://github.com/aio-libs/multidict/issues/926 and isn't related to the pull request.

**Comment by jan-cerny** [comment]
I have rebased this PR on the top of the latest upstream master branch.

**Review by marcusburghardt — CHANGES_REQUESTED** [review]
Nice work. I only have some few comments to take a look.

**Comment by jan-cerny** [comment]
I have done changes according to @marcusburghardt's comments.

File: controls/cis_rhel7.yml [hunk 1]
```diff
@@ -2035,340 +1862,433 @@ controls:
...
       - timer_logrotate_enabled
 
-  - id: 5.1.1
-    title: Ensure cron daemon is enabled and running (Automated)
+  - id: 5.1.4
+    title: Ensure all logfiles have appropriate access configured (Automated)
     levels:
     - l1_server
     - l1_workstation
+    status: partial
```
Comment (marcusburghardt) [inline controls/cis_rhel7.yml:1870]: The status now can be updated to `automated` and the commend can be removed.

**Comment by jan-cerny** [comment]
I have updated the status and removed the comment.

**Review by marcusburghardt — APPROVED** [review]
Great. Thanks for the nice work @jan-cerny . I will wait the CI tests to confirm everything is good and merge it.

**Comment by marcusburghardt** [comment]
I checked the failed Automatus tests and they are failing during the clean-up, after the rule tests.
There are no issues testing the rules, so the tests can be safely waived in this particular case.

We are investigating the issue with automatus task "Delete datastream artifact".

**Pull request merged**